### DISCUSSION TITLE
Error: Could not resolve entry module (rollup.config.js)

### DISCUSSION THREAD

**Discussion by nicky132** [maplibre/maplibre-gl-js#2007] [Q&A]
**maplibre-gl-js version**:2.1.9

**browser**:

### Steps to Trigger Behavior

 1.
 2.
 3.

### Expected Behavior
post out rollup.config.js file
### Actual Behavior
<img width="643" alt="image" src="https://user-images.githubusercontent.com/34881461/209618857-777ad55e-2265-44c0-9b8e-9484193b077a.png">

deploy failed

**Comment by HarelM** [comment]
Please don't repost an issue I already answered.
Please use npm and not yarn.

**Comment by wipfli** [comment]
Issue number 2000!

**Comment by nicky132** [comment]
> Please don't repost an issue I already answered. Please use npm and not yarn.
All right,thank you,by the way ,would you like show me which issue then?

**Comment by HarelM** [comment]
#1582 - you posted the same comment as here and I replied there...
If my answer solves your issue, please close this one.
I would also recommend using a discussion and not an issue if you are not sure what to do or how to use this library.
Bear in mind that every PR is passing through a CI pipline that builds it from scratch to make sure it is working as expected.
If the docs are not great and needs some improvements, feel free to submit a PR to improve the docs.

**Comment by pgoldberg** [comment]
For a production build:

```bash
npm install
npm run build-prod
```

dev build:
```bash
npm install
npm run build-dev
```

**Comment by nicky132** [comment]
Oh,sorry,i found out i made a mistake,my project has not the file,rollup.config.js,but your project has it already,that‘s my problem

**Comment by nicky132** [comment]
>


> 



> For a production build:
> 
> ```shell
> npm install
> npm run build-prod
> ```
> 
> dev build:
> 
> ```shell
> npm install
> npm run build-dev
> ```

Ok,finally,build successfully,and got the dist file and maplibre-gl-dev.js,but how can i use the maplibre-gl-dev.js in html.
Because i used it in index.html already,but i got nothing show up
![image](https://user-images.githubusercontent.com/34881461/209749883-18e7df04-547c-4af6-a6b7-9712e3499eca.png)
